Mathlib/Geometry/Manifold/ChartedSpace.lean

```diff
@@ -4,6 +4,7 @@ Released under Apache 2.0 license as described in the file LICENSE.
 Authors: Sébastien Gouëzel
 -/
 import Mathlib.Topology.PartialHomeomorph
+import Mathlib.Topology.Connected.PathConnected
```

Comment:
Pointing out for later reviewers: this is where the import increase is coming from.